src/Database/DBStructure.php

```diff
@@ -1601,7 +1601,7 @@ public static function definition() {
 						"uid_created" => array("uid","created"),
 						"uid_commented" => array("uid","commented"),
 						"uid_wall_created" => array("uid","wall","created"),
-						"private_wall_received" => array("private","wall","received"),
+						"private_wall_commented" => array("private","wall","commented"),
```

Comment:
Please add a description in friendicas db documentation

Reply:
We don't describe indexes, don't we?

Reply:
Ohh sry. My fault